dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageCheckerArguments.java

```diff
@@ -190,47 +184,15 @@ private static Options configureOptions() {
     }
   }
 
-  /**
-   * Returns a list of absolute paths to jar files for the option. The list includes dependencies if
-   * BOM or Maven coordinates are specified in the option.
-   */
-  ImmutableList<Path> getInputClasspath() throws RepositoryException {
-    if (cachedInputClasspath != null) {
-      // Avoid unnecessary dependency resolution, which Cloud OSS BOM takes around 4 seconds
-      return cachedInputClasspath;
-    }
-
-    if (commandLine.hasOption("b") || commandLine.hasOption("a")) {
-      List<Artifact> artifacts = getArtifacts();
-      ClassPathResult result = classPathBuilder.resolve(artifacts);
```

Comment:
Now it's LinkageCheckerMain's responsibility to resolve the class path through dependency graphs.